src/components/Upload.vue

```diff
@@ -1,62 +1,157 @@
+        <div class="upload_head">
+          <img src="@/assets/upward-arrow.png" />
+          <span class="h2 font-weight-bold align-top">Upload</span>
+        </div>
+
+        <!-- Icons  -->
+        <div class="icons">
+          <label
+            @click="openFileInput()"
+            :v-model="file"
+            class="btn btn-icon-only"
+            for="inputfile"
+            type="button"
+            title="doc button"
+          >
+            <img src="@/assets/doc.png" alt="Document" />
+          </label>
+
+          <label
+            @click="openFileInput"
+            class="btn btn-icon-only"
+            for="inputfile"
+            type="button"
+            title="gallery button"
+          >
+            <img src="@/assets/gallery.png" alt="gallery" />
+          </label>
 
-            <button class="btn btn-icon-only" type="button" title="pdf button">
-              <img src="@/assets/pdf.png" alt="pdf" />
-            </button>
+          <label
+            @click="openFileInput"
+            class="btn btn-icon-only"
+            for="inputfile"
+            type="button"
+            title="pdf button"
+          >
+            <img src="@/assets/pdf.png" alt="pdf" />
+          </label>
 
-            <button class="btn btn-icon-only" type="button" title="text button">
-              <img src="@/assets/text.png" alt="text" />
-            </button>
+          <label
+            @click="openFileInput"
+            class="btn btn-icon-only"
+            for="inputfile"
+            type="button"
+            title="text button"
+          >
+            <img src="@/assets/text.png" alt="text" />
+          </label>
 
-            <button
-              class="btn btn-icon-only"
-              type="button"
-              title="attach button"
-            >
-              <img src="@/assets/attachment1.png" alt="attach" />
-            </button>
+          <label
+            @click="openFileInput"
+            class="btn btn-icon-only"
+            for="inputfile"
+            type="button"
+            title="attach button"
+          >
+            <img src="@/assets/attachment1.png" alt="attach" />
+          </label>
 
-            <button
-              class="btn btn-icon-only"
-              type="button"
-              title="rozprava button"
-            >
-              <img src="@/assets/rozprava.png" alt="rozprava" />
-            </button>
-          </div>
+          <label
+            @click="openFileInput()"
+            class="btn btn-icon-only"
+            for="inputfile"
+            type="button"
+            title="rozprava button"
+          >
+            <img src="@/assets/rozprava.png" alt="rozprava" />
+          </label>
         </div>
       </div>
+    </div>
   </div>
 </template>
 
 
 <script>
+
 export default {
   name: "Upload",
+  props:['files'],
+  data() {
+    return {
+      docfile: false,
+      file: [],
```

Comment:
If it's an array, the variable should be plural too.

Reply:
files are passing in prop so I differentiate it!